src/MainOpt.cxx

```diff
@@ -49,16 +49,19 @@ int MainOpt::parse_config_json(std::string json) {
   if (auto o = get_int(&d, "status-master-interval")) {
     status_master_interval = o.v;
   }
-  if (auto o = get_object(d, "kafka")) {
```

Comment:
Where the options below the `kafka` key previously used for anything else than `stream-master` ?

Reply:
I don't remember was used for anything else; going back in the commit history nothing seems to be using it